### PR TITLE
Manage screen menu items per type of building

### PR DESCRIPTION
**Pull request by javasia**

### CODE REVIEW

**Comment by javasia** [comment]
[Create Page to manage PlaceCategories](https://trello.com/c/RTj9d0Zl/154-create-page-to-manage-placecategories)

File: frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+import ResetButton from './components/Buttons/Reset'
+import './index.scss'
+
+const rows = [
+  { id: 'name', numeric: false, disablePadding: false, label: 'Name' },
+  { id: 'menuItems', numeric: false, disablePadding: false, label: 'menuItems' },
+  { id: 'delete', numeric: false, disablePadding: false, label: 'Delete' },
+];
+
+class EnhancedTableHead extends React.Component {
```
[the remark this answers is not in the document]
Comment (zozich) [reply frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js]: Его вообще можно почистить, и часть этих компонентов удалить

[the remark this answers is not in the document]
Comment (javasia) [reply frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js]: done

File: frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  },
+  actions: {
+    color: theme.palette.text.secondary,
+  },
+  title: {
+    flex: '0 0 auto',
+  },
+});
+
+let EnhancedTableToolbar = props => {
```
[the remark this answers is not in the document]
Comment (javasia) [reply frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js]: done

File: frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/index.js [hunk 3]
```diff
@@ -2,31 +2,18 @@ import React, {Component} from 'react'
...
 import {withRouter} from 'react-router'
 import {menuItemsOperations} from 'store/menuItems'
-import MultipleSelect from './MenuItemMultipleSelect'
-import PlaceCategories from './PlaceCategories'
+import Table from './components/Table'
 import './index.scss'
-import SubmitButton from './SubmitButton'
-import Preloader from '../../../../components/Preloader'
 
 class ScreenMenuItems extends Component {
```
[the remark this answers is not in the document]
Comment (javasia) [reply frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/index.js]: deleted

File: frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+  },
+  table: {
+    minWidth: 1020,
+  },
+  tableWrapper: {
+    overflowX: 'auto',
+  },
+});
+
+class EnhancedTable extends React.Component {
```
[the remark this answers is not in the document]
Comment (javasia) [reply frontend/admin_panel/src/pages/admin/components/ManagePlaceCategories/components/Table/index.js]: Cannot rename it right now (due to out of PC). Hopefully that minor flaw won't block PR.  :)

**Comment by javasia** [comment]
[Create Page to manage PlaceCategories](https://trello.com/c/RTj9d0Zl/154-create-page-to-manage-placecategories)

**Pull request merged**